lib/core/network/models/dao_data_model.dart

```diff
@@ -0,0 +1,26 @@
+class DaoData {
+  final String docId;
+  final String detailsDaoName;
+  final String settingsDaoTitle;
+  final String settingsLogo;
+  final String settingsDaoUrl;
+
+  DaoData({
+    required this.docId,
+    required this.detailsDaoName,
+    required this.settingsDaoTitle,
+    required this.settingsLogo,
+    required this.settingsDaoUrl,
+  });
+
+  factory DaoData.fromJson(Map<String, dynamic> json) {
+    final Map<String, dynamic> settings = json['settings'][0];
+    return DaoData(
+      docId: json['docId'],
+      detailsDaoName: json['details_daoName_n'],
```

Comment:
who named these params??...   details_daoName_n 
Camel + snake what a mess lol

Reply:
Lol - you are right, I will create an update.

Reply:
It was ChatGPT...

Reply:
Actually - wait on our end the names are cool - the AI did the right thing

We can't change the names we get from the service obviously... they are messy.